src/vivarium/framework/lookup.py

```diff
@@ -110,21 +149,22 @@ def setup(self, builder):
             raise ValueError('Only order 0 and order 1 interpolations are supported. '
                              f'You specified {self._interpolation_order}')
 
-    def build_table(self, data, key_columns, parameter_columns):
-        """Construct a TableView from a ``pandas.DataFrame``. An interpolation
-        function of the specified order will be calculated for each permutation
-        of the set of key_columns. The columns in parameter_columns will be used
-        as parameters for the interpolation functions which will estimate all
-        remaining columns in the table.
+    def build_table(self, data, key_columns, parameter_columns, value_columns):
+        """Construct a LookupTable from input data.
```

Comment:
This documentation should really be on the ``LookupTableInterface`` since that's the public interface.